roles/network_plugin/cilium/templates/cilium-deploy.yml.j2

```diff
@@ -84,7 +84,7 @@ spec:
                   key: AWS_DEFAULT_REGION
                   name: cilium-aws
                   optional: true
-          image: docker.io/cilium/operator:v1.5.5
+          image: "{{cilium_image_repo}}:{{cilium_operator_image_tag}}"
```

Comment:
write var name with space in brackets
as "{{ cillium_image_repo }}"

Reply:
Fixed